pombola/hansard/management/commands/hansard_check_for_new_sources.py

```diff
@@ -66,9 +72,16 @@ def process_url(self, list_page, url, verbose):
             convertEntities=BeautifulStoneSoup.HTML_ENTITIES
         )
 
-        spans = soup.findAll( 'span', 'contenttype-repositoryitem summary')
+        base_url = '/'.join(url.split('/')[:3])
```

Comment:
I found this a bit unclear - it's probably simpler to use [the urlparse module](https://docs.python.org/2/library/urlparse.html) for this, I think, e.g.

```python
from urlparse import urlsplit, urlunsplit
url_parts = urlsplit(url)
base_url = urlunsplit((url_parts.scheme, url_parts.netloc, '', '', ''))
```